examples/data_masking/sam/template.yaml

```diff
@@ -0,0 +1,59 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  Powertools for AWS Lambda (Python) data masking example
+
+Globals: # https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-specification-template-anatomy-globals.html
+  Function:
+    Timeout: 5
+    Runtime: python3.11
+    Tracing: Active
+    Environment:
+        Variables:
+          POWERTOOLS_SERVICE_NAME: PowertoolsHelloWorld
+          POWERTOOLS_METRICS_NAMESPACE: Powertools
+          LOG_LEVEL: INFO
+          KMS_KEY_ARN: !GetAtt DataMaskingKMSKey.Arn
+
+
+Resources:
+  # Lambda Function
```

Comment:
Explain that this is for documentation purposes; in prod, we recommend splitting Encrypt and Decrypt, so one function can act as encryption proxy (HTTP reqs, Data pipeline, etc) while authorized personnel can either call Decrypt from scripts... OR from a separate function altogether.

Same with IAM permissions, we need an example for Decrypt only